server/game/core/ongoingEffect/OngoingEffectEngine.ts

```diff
@@ -75,7 +75,7 @@ export class OngoingEffectEngine {
                     }
                     const actionEvents = [];
                     properties.gameAction.queueGenerateEventGameSteps(actionEvents, context);
-                    this.game.queueSimpleStep(() => this.game.openThenEventWindow(actionEvents), 'openAdditionThenEventWindow');  // TODO: why is it using this window type?
+                    this.game.queueSimpleStep(() => this.game.openEventWindow(actionEvents), 'openAdditionThenEventWindow');  // TODO: why is it using this window type?
```

Comment:
Can you do me a favor and change the step name to be `'openDelayedActionsWindow'` and also remove the comment, I know what this is doing now